stylesheets/screen.css

```diff
@@ -552,30 +556,9 @@ input[type="search"]:focus::-webkit-search-cancel-button::after{
 .page{ padding-top: 1em; }
 
 .page .wrapper{
-	width: auto;
-	max-width: none;
-}
-
-.page .page-title,
-.page h1,
-.page h2,
-.page h3,
-.page h4,
-.page h5,
-.page nav,
-.page figure,
-.page p,
-.page ul,
-.page ol,
-.page dl,
-.page li{
-	line-height: 1.4em;
-	margin: 0 auto 0.7em;
-	max-width: 640px;
-	width: 90%;
+	max-width: 960px;
```

Comment:
These are a lot of classes/elements affected by this; I took a look at the page in the PR to just poke around a bit and make sure things still looked okay, which they appear to, so just calling it out to be sure!

Reply:
Thanks for calling this out @ccostino - these elements are receiving multiple styles overriding styles, overriding styles so i determined these were ok to clean up. _Sidenote_ The styles could really be streamlined - I wasn't expecting over 1000 lines of CSS :)

Reply:
Yeah I can imagine - I didn't put all of this together, so I wasn't expecting that either... 😆 Thanks for making the effort to clean it up and improve it little by little though! :-)